Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -208,6 +205,17 @@ WebsiteDataStore* WebsiteDataStore::existingDataStoreForSessionID(PAL::SessionID
     return allDataStores().get(sessionID);
 }
 
+#if HAVE(APP_SSO)
+SOAuthorizationCoordinator* WebsiteDataStore::soAuthorizationCoordinator(const WebPageProxy& pageProxy)
```

Comment:
This function never returns null so it should probably return a reference, not a raw pointer. Also, this means we don't need the null checks at call sites.